sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -451,9 +451,13 @@ export class BatchingReceiver extends MessageReceiver {
    * @param {ClientEntityContext} context    The connection context.
    * @param {ReceiveOptions} [options]     Receive options.
    */
-  static create(context: ClientEntityContext, options?: ReceiveOptions): BatchingReceiver {
+  static create(
+    context: ClientEntityContext,
+    options?: ReceiveOptions,
+    retryOptions?: RetryOptions
+  ): BatchingReceiver {
     throwErrorIfConnectionClosed(context.namespace);
-    const bReceiver = new BatchingReceiver(context, options);
+    const bReceiver = new BatchingReceiver(context, options, retryOptions);
```

Comment:
Why make changes to the batching receiver at all? Retries for the receive batch feature is handled on the outside as per #8142 8136

Reply:
My only reason was to make the constructors look similar for Batching and Streaming Receivers. Though they are being passed, it doesn't affect the Batching receiver.


Now, I have moved the retryOptions to the options as you said, and not passing the retryOptions for the Batching receiver when it is being created.